lib/index.js

```diff
@@ -1,5 +1,60 @@
-// This file is in the entry point in your webpack config.
+const getSenators = () => {
```

Comment:
It may be worth moving this code into separate files and then requiring those files here with something like 
```
require('path/to/file')
```
This would allow you to better organize your code.